andela/hc-rogue-one
# FT 156577839 Add periodic reports

#### What does this PR do?
Allow a user to set periodic intervals after which to receive reports about a check.
#### Description of Task to be completed?
User should be able to set periods either monthly or weekly or daily, after which they should receive reports about a check.
#### How should this be manually tested?
1. Install the project and install the requirements.txt
2.Run the commands below
3.You should have send email configured 
```
cd to hc-rogue-one
python manage.py shell
>>> from django.contrib.auth.models import User
>>> from hc.accounts.models import Member, Profile
>>> from hc.api.models import Check
>>> peter=User(username='peter',email='peterwade153@gmail.com')
>>> profile=Profile(user=peter,api_key='abc')
>>> check=Check(name='test',user=peter)
>>> peter.save()
>>> profile.save()
>>> check.save()
>>> peter.profile.send_report()
```
#### What are the relevant pivotal tracker stories?
#156577839

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
